src/resources/Enums.ts

```diff
@@ -730,6 +730,28 @@ export enum StatementGroupType {
     permanent = 'permanent',
 }
 
+/**
+ * An union of all the possible status that you can filter on.
+ */
+export enum ListStatementGroupStatusType {
```

Comment:
I could try to merge the enums here ([as seen here](https://stackoverflow.com/questions/48478361/how-to-merge-two-enums-in-typescript)) but it introduce some constraints that we don't necessary have on the backend. So for now I'm using a different enum for the parameter.